src/templates/static/home.jsx

```diff
@@ -350,6 +350,10 @@ const Home = () => {
                                 <img className='responsive' src={it.url_for('images/pages/home/awards/gbm-2018.svg')} alt='Global Brands Magazine 2018' />
                                 <span className='caption'>{it.L('Global Brands Magazine')}</span>
                             </div>
+                            <div className='award-icon'>
+                                <img className='responsive' src={it.url_for('images/pages/home/awards/gbm-2019.svg')} alt='Global Brands Magazine 2019' />
```

Comment:
```suggestion
                                <img className='responsive' src={it.url_for('images/pages/home/awards/gbm-2019.svg')} alt={it.L('Global Brands Magazine 2019')}/>
```

Reply:
in binary static, no alt tag for image is being translated actually.

Reply:
why not?

Reply:
im not sure why 😅 .

Reply:
probably we are removing translation workload here

Reply:
I see well I think that comes at the expense of SEO and accessibility

Reply:
ok, lets create a card to add translation for all alt image in binary static